modules/end-user-guide/partials/proc_configuring-an-existing-workspace-to-use-intellij-idea.adoc

```diff
@@ -0,0 +1,21 @@
+[id="configuring-an-existing-workspace-to-use-intellij-idea_{context}"]
+= Configuring an existing workspace to use IntelliJ IDEA
+
+You can configure an already existing workspace to use IntelliJ IDEA as the editor in the {prod-short} dashboard.
+
+.Procedure
+
+. Navigate to the {prod-short} dashboard -> *Workspaces* ->  <your_workspace_name> -> *Devfile*.
+
+. Add the following section to the workspace devfile:
```

Comment:
```suggestion
. Add the following component to the workspace devfile:
```

we need to make it clear that shared fragment should be put into `components` field.